### ISSUE TITLE
Built-in verb synonyms take precedence over meta commands for the same input

### ISSUE THREAD

**Issue by DavidKinder** [DavidKinder/Windows-AGiliTy#8]
In the Multi-Dimensional Thief, a creature called "OUT" is defined so that "GET OUT" is a valid command, and the game includes a meta command to process this in one location (in the water at the waterfall). But AGiliTy defines "GET OUT" as a synonym for "OUT", so by the time it gets to the meta command processing stage, the input "GET OUT" has already been rewritten to "OUT", so the game's meta command for "GET OUT" is never run.

**Comment by DavidKinder** [comment]
The root of the problem is that the metacommand for "GET OUT" is stored internally as applying to a verb of "GET" and a noun of "OUT", but when "GET OUT" is entered, it is converted (via the built-in list of synonyms) to just "OUT", so the "GET OUT" metacommand is never run. MDT contains another of these, a metacommand for "LIST EXITS".

The solution I've adopted is to have the AGT loader rewrite these metacommands into the appropriate form that will be matched by the parser. So the "GET OUT" metacommand is rewritten to apply to a verb of "OUT" without a noun, which then works.

One consequence of this change is that it is necessary to sort the table of metacommands, so that the modified metacommand ends up in the right place. Previously AGiliTy would only do this if loading not from an AGX file, but with the output of the MAGX compiler. I did consider only doing the sort if I rewrote a metacommand, but I've decided that it's cleaner and simpler to just always do the sort. The sort should never break anything, and if it does, we should fix it.
